public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -168,11 +180,9 @@ class UnConnectedExploreToolbar extends PureComponent<Props> {
             tooltip={`${isLargerExploreId ? 'Narrow' : 'Widen'} pane`}
             disabled={isLive}
             onClick={onClickResize}
-            icon={
-              (exploreId === 'left' && isLargerExploreId) || (exploreId === 'right' && !isLargerExploreId)
-                ? 'angle-left'
-                : 'angle-right'
-            }
+            icon={isLargerExploreId ? 'gf-movepane-right' : 'gf-movepane-left'}
+            iconOnly={true}
+            className={styles.rotateIcon}
```

Comment:
as styles would get recreated and inserted at every render, can't we just switch the icon based on `exploreId` instead of rotating it?

Reply:
The idea here is to keep the arrow in the pane where it's displayed in the icon. So the left pane always has an arrow on the left side, and the icon itself either has the arrow pointing into or away from the smaller pane depending on if we're making the pane larger or not.

Reply:
![Screenshot 2022-12-20 at 13 28 19](https://user-images.githubusercontent.com/1170767/208678157-179e3586-a92e-4dfa-9b5c-c172d9e554df.png)

that doesn't seem to be the case though 🤔 

also the arrows in the new icons always points internally, if you rotate one of the icons by 180 deg you simply get the other icon

Reply:
Apologies @gelicia , I didn't pick up on this detail but I think @Elfo404 is correct.  For reference here are the previous icons (below) and the latest (above).
<img width="722" alt="Screenshot 2022-12-20 at 9 23 43 AM" src="https://user-images.githubusercontent.com/3299847/208689718-6ff147ca-a7af-4ee9-86e6-1bd2bdd1354a.png">

Reply:
I modified it if that helps? 
![movepane-rtl(1)](https://user-images.githubusercontent.com/3299847/208690374-ce9478a4-4b89-4e52-ae84-6487ee812c5e.svg)

Reply:
> ![Screenshot 2022-12-20 at 13 28 19](https://user-images.githubusercontent.com/1170767/208678157-179e3586-a92e-4dfa-9b5c-c172d9e554df.png)
> 
> that doesn't seem to be the case though 🤔
> 
> also the arrows in the new icons always points internally, if you rotate one of the icons by 180 deg you simply get the other icon

@radiorental @Elfo404 Ah sorry, I didn't know the assets didn't include both arrows. Fixing.